editors/emacs/Portfile

```diff
@@ -211,10 +196,14 @@ if {$subport eq $name || $subport eq "emacs-devel"} {
         depends_lib-append       port:dbus
     }
 
+    variant xwidgets requires gtk description {Enable use of xwidgets in Emacs buffers} {
```

Comment:
xwidgets does not require GTK. It works with AppKit now.

Reply:
Doesn't it require GTK if you're not building the NS app? The -app subports have a separate variant declaration that doesn't require GTK.

Reply:
Oh right, my mistake.